test/suites/ekuiper/main_test.go

```diff
@@ -54,13 +61,20 @@ func TestMain(m *testing.M) {
 	utils.SnapStop(nil, "edgex-ekuiper")
 	utils.SnapSet(nil, "edgexfoundry", "security-secret-store", "off")
 	utils.SnapSet(nil, "edgex-ekuiper", "edgex-security", "off")
+	utils.SnapSet(nil, "edgex-device-virtual", "app-options", "true")
+	utils.SnapSet(nil, "edgex-device-virtual", "config.edgex-security-secret-store", "false")
 	utils.Exec(nil, "sudo rm /var/snap/edgex-ekuiper/current/edgex-ekuiper/secrets-token.json")
 	utils.SnapDisconnect(nil,
 		"edgexfoundry:edgex-secretstore-token",
 		ekuiperSnap+":edgex-secretstore-token",
 	)
+	utils.SnapDisconnect(nil,
+		"edgexfoundry:edgex-secretstore-token",
+		deviceVirtualSnap+":edgex-secretstore-token",
+	)
```

Comment:
When `config.edgex-security-secret-store` is set to false, the token isn't read. We don't need to disconnect.